### DISCUSSION TITLE
Grab Errors and warnings from BMS

### DISCUSSION THREAD

**Discussion by s-hufschmid** [syssi/esphome-seplos-bms#57] [General]
Would it be possible to grab also warning & error messages from BMS like in the battery monitor ?
And show it on HA ?

**Comment by syssi** [comment]
The decoding of warnings and errors isn't implemented yet. You could subscribe this issue (#6) to get informed about any progress here.

**Comment by s-hufschmid** [comment]
Servus!

Vielen Dank für den Hinweis. Habe das issue abonniert.

VG

Stefan

Von: Sebastian Muszynski ***@***.***>
Datum: Dienstag, 4. Juli 2023 um 08:18
An: syssi/esphome-seplos-bms ***@***.***>
Cc: werbung ***@***.***>, Author ***@***.***>
Betreff: Re: [syssi/esphome-seplos-bms] Grab Errors and warnings from BMS (Discussion #57)

The decoding of warnings and errors isn't implemented yet. You could subscribe this issue (#6<https://github.com/syssi/esphome-seplos-bms/issues/6>) to get informed about any progress here.
